docs/paid-features/overview.md

```diff
@@ -34,7 +34,7 @@ Paid plans include more ways to manage permissions, including data sandboxing, w
 - [Download permissions](../permissions/data.md#download-results)
 - [Connection impersonation](../permissions/data.md#impersonation-access)
 - [Database management permissions](../permissions/data.md#manage-database)
-- [Data model management permissions](../permissions/data.md#manage-data-model)
+- [Table metadata management permissions](../permissions/data.md#manage-table-metadata)
```

Comment:
This seems unrelated to the naming change, but I guess it makes sense? I don't have the context so I don't know.

Reply:
We renamed the admin "Data model" tab to "Table metadata" to distinguish it from models. Just say this link issue when reviewing this PR, so I tacked the change on here while I was at it.